SCF-Session-6/polynomial/PolyTest.java

```diff
@@ -0,0 +1,95 @@
+package polynomial;
+
+import static org.junit.Assert.*;
+
+public class PolyTest {
+
+    /**
+     * Test case when input polynomial is null
+     */
+    @org.junit.Test(expected = AssertionError.class)
+    public void polyFirstException() 
+    {
+        new Poly(null);
+    }
+    
+    
+    /**
+     * Test case when input polynomial is empty
+     */
+    @org.junit.Test(expected = AssertionError.class)
```

Comment:
Import `org.junit.Test` package instead of using `@org.junit.Test` for each test method.

Reply:
done